logstash-core/lib/logstash/util/wrapped_acked_queue.rb

```diff
@@ -332,10 +332,6 @@ def flush_signal_received?
 
       private
 
-      def iterating?
-        @iterating
-      end
```

Comment:
May I suggest that you create a discuss issue to use prefix and progressively clean up `name?` methods. Especially as you can explain the JRuby inlining problem just once for all to read. When I say all (I have to repeat what Jordan would say) I also mean the many non-LS folk that follow what we do, that may not read every PR. These external people may have custom plugins or forks of core ones.